database/dbMethods.js

```diff
@@ -1,38 +1,34 @@
-const client = require('../database/index');
+/* eslint-disable no-console */
+const client = require('./index');
 
 module.exports = { 
   getOne: (selector) => {
-    return client.query('select * from stockinfo where stockid=$1', [selector])
+    return client.query(
+      'select * from stockinfo where stockid=$1',
+      [selector])
     .then((res) => {
       return res.rows[0];
     })
     .catch(e => console.error(e.stack));
   },
   
-  updateOne: () => {
-    const query = 'update stockinfo set changePercent=5.11 where stockId=$1';
-    const selectors = ['PLUG000000001'];
-    client.query(query, selectors)
-    .then(res => {
-      console.log('Updated!');
-    })
-    .then(() => client.end())
+  updateOne: (selector) => {
+    return client.query(
+      'update stockinfo set changePercent=5.11 where stockId=$1',
```

Comment:
i'm a little lost on the 5.11 here & the values on line 31. Are these hardcoded? If so, were they just used for testing these routes? If that's the case you may want to consider moving them to a separate file to improve readability